### PR TITLE
Gagan3012

### PR DESCRIPTION
**Pull request by gagan3012**
image

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -958,5 +958,12 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/58569950/120160642-14db8f80-c214-11eb-8f2a-d5946aa8d59e.png",
     "issueNumber": 537
-  }
+  },
+  {
+    "name": "Gagan Bhatia",
+    "githubUsername": "gagan3012",
+    "imageUrl": "https://user-images.githubusercontent.com/49101362/120364787-7c501880-c2c2-11eb-8f18-0f0ef10c4439.png",
+    "issueNumber": 539
+  },
```
Comment (gagan3012) [inline data.json:967]: Remove this comma

File: data.json [hunk 2]
```diff
@@ -958,5 +958,12 @@
...
+
```
Comment (schmelto) [inline data.json:968]:
```suggestion
```

File: data.json (hunk 1 above)
Comment (schmelto) [inline data.json:967]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by schmelto** [comment]
Could you also add `resolve #yourissuenumber ` to the pr description?

**Comment by eddiejaoude** [comment]
Did you mean to close this @gagan3012 ?

**Comment by eddiejaoude** [comment]
Oh I see there is another PR https://github.com/EddieHubCommunity/awesome-github-profiles/pull/541

**Pull request closed without merging**